data-migrator/qa/integration-tests/src/test/java/io/camunda/migration/data/qa/history/HistoryMigrationAbstractTest.java

```diff
@@ -72,6 +72,9 @@ public abstract class HistoryMigrationAbstractTest extends AbstractMigratorTest
   @Autowired
   protected RdbmsService rdbmsService;
 
+  @Autowired
+  protected io.camunda.migration.data.impl.clients.C8Client c8Client;
+
```

Comment:
This will trigger architecture tests to fail, see [here](https://github.com/camunda/camunda-7-to-8-migration-tooling/pull/796/changes#r2642749750)